README.md

```diff
@@ -43,6 +43,15 @@ For more information, please refer to the detailed documentations below.
 | [Alerts](./docs/src/pages/guides/alerts.md)              | How alerts are triggered and how to setup an alert |
 | [Notifcations](./docs/src/pages/guides/notifications.md) |  Receive notifications when alerts are triggered   |
 
+## Run Monika using Docker
+
+Obtain a copy of your configuration file `config.json` into the current directory.
+From the same directoy, you can run:
+
+```
+docker run --name monika -v ${PWD}/config.json:/config/config.json --detach hyperjump/monika:latest
+```
+
```

Comment:
Maybe also shows the command to view the logs from the docker container.

Reply:
If they use docker, they should've know how to see docker log.

Reply:
Please still add it. It's not a difficult thing to do, and if they forgot how to do it, we can save them time. 😃

Reply:
The thing is, its a docker subject. and expect the user to know how to operate docker prior interested to use monika docker, if not, they should've not used docker.

docker logs <container name> is so trivial, that you dont have to explain to user. its like explaining how to install npm to develop node app. 

if we entertain this, there will be question "how to stop ?" "how to remove image?" "how to run in non detach?" 
'how to auto start when docker start?"